ACPs/23-p-chain-native-transfers.md

```diff
@@ -0,0 +1,58 @@
+```text
+ACP: <PR Number>
+Title: P-chain native transfers
+Author(s): Dhruba Basu <https://github.com/dhrubabasu>
+Discussions-To: <GitHub Discussion URL>
+Status: Proposed
+Track: Standards
+```
+
+## Abstract
+
+Support native transfers on P-chain.
+
+## Motivation
+
+Currently, the P-chain has no simple transfer transaction type. The X-chain supports this functionality through a `BaseTx`. Although the P-chain contains transaction types that extend `BaseTx`, the `BaseTx` transaction type itself is not a valid transaction.
+
+This ACP details the functionality of the new transaction type. Activation of `BaseTx` on the Avalanche Network must be done in a mandatory upgrade.
+
+## Specification
```

Comment:
I think for this section we should be pretty clear/verbose about what the behavior of `BaseTx`. What does it mean to support a `BaseTx`? I'm not sure if we've chatted about what we want ACP's to look like but I feel like ideally people implementing their own clients should be able to fully support an ACP without having to reference anything other than this spec.

Reply:
Updated- I think this one's a little odd since `BaseTx` should already be implemented in all Avalanche clients as all other transaction types in the P-chain extend it. Added the concrete typeIDs though